components/edit-collective/sections/InviteMemberModal.js

```diff
@@ -0,0 +1,190 @@
+    $role: MemberRole!
+    $description: String
+    $since: DateTime
+  ) {
+    inviteMember(
+      memberAccount: $memberAccount
+      account: $account
+      role: $role
+      description: $description
+      since: $since
+    ) {
+      id
+      role
+      description
+      since
+    }
+  }
+`;
+
+const InviteMemberModal = props => {
+  const { intl, show, collective, membersIds, cancelHandler } = props;
+
+  const { addToast } = useToasts();
+
+  const [member, setMember] = React.useState(null);
+  const mutationOptions = {
+    context: API_V2_CONTEXT,
+    refetchQueries: [
+      {
+        query: coreContributorsQuery,
+        context: API_V2_CONTEXT,
+        variables: {
+          collectiveSlug: get(collective, 'slug'),
+          account: { slug: get(collective, 'slug') },
+        },
+      },
+    ],
+    awaitRefetchQueries: true,
+  };
+
+  const [inviteMemberAccount, { loading: isInviting, error: inviteError }] = useMutation(
+    inviteMemberMutation,
+    mutationOptions,
+  );
+
+  let submitMemberForm = null;
+
+  const bindSubmitForm = submitForm => {
+    submitMemberForm = submitForm;
+  };
+
+  const handleInviteMemberMutation = async values => {
+    const { description, role, since } = values;
+
+    try {
+      await inviteMemberAccount({
+        variables: {
+          memberAccount: {
+            slug: get(member, 'slug'),
+          },
+          account: { slug: get(collective, 'slug') },
+          description,
+          role,
+          since,
+          isInvitee: true,
+        },
+      });
+
+      addToast({
+        type: TOAST_TYPE.SUCCESS,
+        message: <FormattedMessage id="editTeam.member.invite.success" defaultMessage="Member invited successfully." />,
+      });
+
+      cancelHandler();
+    } catch (error) {
+      addToast({
+        type: TOAST_TYPE.ERROR,
+        message: <FormattedMessage id="editTeam.member.invite.error" defaultMessage="Failed to invite member." />,
+      });
+    }
+  };
+
+  const handleSubmitForm = () => {
+    if (submitMemberForm) {
+      submitMemberForm();
+    }
+  };
+
+  return (
+    <Container>
+      <Modal width={688} show={show} onClose={cancelHandler}>
+        <ModalHeader mb={4}>
+          <FormattedMessage id="editTeam.member.invite" defaultMessage="Invite Team Member" />
+        </ModalHeader>
+        <ModalBody>
+          {inviteError && (
+            <Flex alignItems="center" justifyContent="center">
+              <MessageBox type="error" withIcon m={[1, 3]} data-cy="cof-error-message">
+                {inviteError.message}
+              </MessageBox>
+            </Flex>
+          )}
+          <Flex m={1} flexDirection="column" mb={2}>
+            <P fontSize="14px" lineHeight="20px" fontWeight={700} mb={1}>
+              <FormattedMessage id="Tags.USER" defaultMessage="User" />
+            </P>
+            <CollectivePickerAsync
+              inputId="member-collective-picker"
+              creatable
```

Comment:
Usually, a single maintainer creates a collective and invites other maintainers of the project after creating. So, this can come in handy if other maintainers do not have an OC account at the moment.

Also, we use it in e2e tests so we will need to update tests as well if we remove this.